ci/matrix.yaml

```diff
@@ -7,8 +7,9 @@ workflows:
   # override:
   #   - {jobs: ['test'], project: 'thrust', std: 17, ctk: 'curr', cxx: ['gcc12', 'clang16']}
   #
-  override:
 
+# Only run python build and test jobs for testing
```

Comment:
These comments don't seem related to this part of the file. Should they be moved / removed?